media/css/mediaelementplayer.css

```diff
@@ -1001,7 +1001,7 @@ div.mejs-speed-button {
 .mejs-poster {
 	text-align: center;
 	background: rgba(0,0,0,0.75);
-	z-index: 1000;
+	z-index: 100000000;
```

Comment:
I don't know if this is a good practice to go so high in the number but it works.

Reply:
Its priority for layers. Now form gated layer has higher priority